bin/pump-import-email

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env node
 
 // -*- mode: javascript -*-
-
+//
+// pump-import-email
+//
 // import an activitystreams JSON collection
```

Comment:
This still seems incorrect. AFAICT from the source it should be something along the lines of "set the email of a user"